packages/elements/src/autosuggest/__demo__/index.html

```diff
@@ -652,5 +652,45 @@
         })();
       </script>
     </demo-block>
+    <demo-block layout="normal" header="Accessibility" tags="accessibility">
+      <label for="accessibility-input">Select Employee</label>
+      <input id="accessibility-input"
+             type="search"
+             aria-controls="accessibility-suggest"
+             aria-expanded="false"
+             aria-autocomplete="both"
+             aria-haspopup="listbox">
+      <ef-autosuggest id="accessibility-suggest"
+                      style="min-width: 300px"
+                      attach="#accessibility-input"></ef-autosuggest>
+      <script>
+        (function () {
+          const autoSuggest = document.getElementById('accessibility-suggest');
+          const input = document.getElementById('accessibility-input');
+          autoSuggest.addEventListener('suggestions-fetch-requested', ({ target, detail: { query } }) => {
+            window.filterData(query).then(({ data }) => {
+              target.suggestions = data;
+            });
+          });
+          autoSuggest.addEventListener('opened-changed', (event) => {
+            const opened = event.detail.value;
+            input.setAttribute('aria-expanded', `${opened}`);
+            if (!opened) {
+              input.removeAttribute('aria-activedescendant');
+            }
+          });
+          autoSuggest.addEventListener('item-select', (event) => {
```

Comment:
I thought this is suppose to be `item-highlight` event, but I'm surprise `item-select` also works similarly.

Reply:
Autosuggest is not really consistent with the rest of framework, so that is one of the reasons why it needs refactoring.

Reply:
You are right, it should be highlight

Reply:
well, no, it needs to be item-select to work correctly

Reply:
What's the difference ? I tried switching to `item-highlight` and it still update id correctly.